OpenSim/Simulation/MarkersReference.cpp

```diff
@@ -77,19 +77,13 @@ void MarkersReference::populateFromMarkerData(const MarkerData& aMarkerData)
     _markerNames.assign(nm, "");
     _weights.assign(nm, get_default_weight());
 
-    int index = 0;
-    // Build flat lists (arrays) of marker names and weights in the same order as the marker data
-    for(int i=0; i<tempNames.getSize(); i++){
+    for (int i = 0; i < tempNames.getSize(); i++) {
         const std::string &name = tempNames[i];
         _markerNames[i] = name;
```

Comment:
Should there be a check here to ensure the specified marker name exists in the model?